app/src/test/java/camp/nextstep/edu/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,94 @@
+package camp.nextstep.edu.calculator
+
+import com.example.calculatorlib.Calculator
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.junit.runners.Parameterized.Parameters
+
+@RunWith(value = Parameterized::class)
+class CalculatorTest(val input: String, val expected: Int) {
+    private lateinit var calculator: Calculator
+
+    @Before
+    fun initCalculator() {
+        calculator = Calculator()
+    }
+
+    @Test
+    fun 문자열_수식을_계산한다() {
+        // when : calculator를 통해 수식을 계산한다.
+        val actual = calculator.evaluate("1 + 2 + 3")
+
+        // then : 사칙 연산의 우선 순위가 아닌 입력 우선 순위로 계산을 진행한다.
+        assertThat(actual).isEqualTo(6)
+    }
+
+    @Test
+    fun 사칙연산_우선순위와_상관없이_문자열의_순차적으로_수식을_계산한다() {
+        // when : calculator를 통해 수식을 계산한다.
+        val actual = calculator.evaluate("10 * 2 + 8 / 3")
+
+        // then : 사칙 연산의 우선 순위가 아닌 입력 우선 순위로 계산을 진행한다.
+        assertThat(actual).isEqualTo(9)
+    }
+
+    @Test
+    fun 여러_문자열_수식을_계산한다() {
+        // when : calculator를 통해 수식을 계산한다.
+        val actual =  calculator.evaluate(input)
+        // then : 사칙 연산의 우선 순위가 아닌 입력 우선 순위로 계산을 진행한다.
+        assertThat(actual).isEqualTo(expected)
+    }
+
+    @Test
+    fun 문자열_수식이_공백일_경우_에러를_발생시킨다() {
+        // when : 수식을 공백 문자열로 채운다.
+        val exception = kotlin.runCatching { calculator.evaluate("        ") }.exceptionOrNull()
```

Comment:
`kotlin.`은 제거하셔도 괜찮습니다 :)
```suggestion
        // when : 수식을 공백 문자열로 채운다.
        val exception = runCatching { calculator.evaluate("        ") }.exceptionOrNull()
```